modules/ROOT/pages/_partials/include-alerts-config.adoc

```diff
@@ -8,37 +8,37 @@ TODO_FUTURE?
 You can create, configure, delete, test, and send alerts to people who need to see them. The rules you configure trigger alerts when conditions reach a specified threshold value, for example: *TODO_FUTURE?*  You can also get the history of an alert.
 ////
 
-image::graph-alert-config.png[Alert Configuration]
+image::graph-alert-config.png[Advanced Alert Configuration]
 
 
 . Navigate to the configuration screen for a graph in a custom dashboard. For guidance, see xref:dashboard-custom-config.adoc[Configuring Custom Dashboards].
-. Provide a General configuration for the graph that (at minimum) defines the metric, environment, and resource for which you want to set the alert. For guidance, see xref:dashboard-custom-config-graph.adoc#general_settings[General Settings].
-. Navigate to the Alert configuration tab for the graph.
+. Provide a general configuration for the graph that (at minimum) defines the metric, environment, and resource for which you want to set the advanced alert. For guidance, see xref:dashboard-custom-config-graph.adoc#general_settings[General Settings].
+. Navigate to the *Alert* configuration tab for the graph.
 . Click *Create Alert*.
 . Use *Alert Config* to configure the alert. See <<alert_config, Alert Config>>.
-. Configure any notifications to email when an alert is triggered. See <<notifications, Notifications>>.
+. Configure any notifications to email when an advanced alert is triggered. See <<notifications, Notifications>>.
```

Comment:
Ditto on all counts of the above.

Consider "to send" rather than "to email."

Reply:
```suggestion
. Configure email notifications when an advanced alert is triggered. See <<notifications, Notifications>>.
```